dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,106 @@
+class SentryStackFrame {
+  static final SentryStackFrame asynchronousGapFrameJson = SentryStackFrame()
+    ..absPath = '<asynchronous suspension>';
+
+  List<String> preContext;
+  List<String> postContext;
+  Map<String, String> vars;
+  List<int> framesOmitted;
```

Comment:
should we add getters to allow mutable list/map?

Reply:
✅  immutable w/ `copyWith`